configs/openSUSE/opensuse.toml

```diff
@@ -0,0 +1,255 @@
+# Configuration for the rpmlint utility.
+
+# Configuration options used by the checks
+CompressExtension = ""
+UseVarLockSubsys = false
+UseVersionInChangelog = false
+BadnessThreshold = 1000
+
+# Enabled checks for the rpmlint to be run (besides the default set)
+Checks = [
+# TODO Uncomment security checks when they are ported
+# Note: polkit-default-privs would need to be installed always
+    #"CheckSUIDPermissions",
+    #"CheckDBUSServices",
+    #"CheckCronJobs",
+    #"CheckPolkitPrivs",
+    #"CheckWorldWritable",
+    #"CheckDeviceFiles",
+
+# TODO: Uncomment in opensuse branch only
+    #"BrandingPolicyCheck",
+    #"FilelistCheck",
+    #"KMPPolicyCheck",
+    #"SystemdInstallCheck",
```

Comment:
Wouldn't this part actually make sense to be just a followup commit on the opensuse branch rather than something commented out in here?